src/App.jsx

```diff
@@ -10,74 +15,65 @@ export const goods = [
   'Bread',
   'Fish',
   'Honey',
-  'Jam',
+  DEFAULT_GOOD,
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(DEFAULT_GOOD);
```

Comment:
For the default value, it is better to refer directly to the array of `goods` to understand that we will always have relevant information.
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```